payjoin-client/src/main.rs

```diff
@@ -25,10 +25,14 @@ fn main() {
         .into_string()
         .expect("bip21 is not UTF-8");
 
-
     let link = bip21.parse::<bip78::Uri>().unwrap();
+
+    if link.amount().is_none() {
+        panic!("please specify the amount in the Uri");
+    }
+
     let mut outputs = HashMap::with_capacity(1);
-    outputs.insert(link.address().to_string(), link.amount());
+    outputs.insert(link.address().to_string(), link.amount().unwrap());
```

Comment:
I would find `link.amount().expect("please specify the amount in the Uri")` cleaner. Not a big deal since I'd like to eventually either make it interactive or allow specifying the amount as a separate argument.